app/models/concerns/bulkrax/export_behavior.rb

```diff
@@ -27,10 +27,10 @@ def hyrax_record
     # Prepend the file_set id to ensure a unique filename and also one that is not longer than 255 characters
     def filename(file_set)
       # return if there are no files on the fileset
-      return if Bulkrax.object_factory.original_file(fileset: file_set).blank?
+      file = Bulkrax.object_factory.original_file(fileset: file_set)
+      return if file.blank?
```

Comment:
Again, if we're expecting a string, let's return an empty string if we don't have the info we need, to keep a more consistent API.

```suggestion
      return '' if file.blank?
```

I know that this is a shift from the Hyrax pattern, but I think it might help us slowly dig ourselves out of the error reporting / saving / rescuing problems we've been having.